scripts/monitoring/cron-send-security-updates-count.py

```diff
@@ -82,7 +82,7 @@ def get_security_updates(self):
             self.sec_updates = []
             raise subprocess.CalledProcessError(proc.returncode, subprocess.list2cmdline(cmd), output=stdout+stderr)
         lineslist = [line for line in stdout.split('\n') if line and 'Errno' not in line]
-        self.sec_updates = [UpdateItem(*re.split(r'\s+', line)) for line in lineslist]
+        self.sec_updates = [UpdateItem(*re.split(r'\s+', line)) for line in lineslist if re.split(r'\s+', line) == 3]
```

Comment:
We came up with this quick fix as a testing solution to understand the problem. I think the proper way to proceed would be to change the list comprehension into a proper loop where if things fail exceptions can be caught and potentially displayed or logged, then proceed with the other security update detection.